src/ai/ai.cpp

```diff
@@ -346,7 +348,15 @@ void ShipAI::runOrders()
                 flyTowards(owner->getOrderTargetLocation());
             }
         }else{
```

Comment:
Yup, it is a tricky situation. In the current code (before this commit), an unarmed vessel going in Roaming mode will do nothing. Roaming mode action conditions and prioritization should be changed. (I also want to make very damaged ships seek repairs after battle).